pages/news/nightstand.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Nightstand mode
+layout: news
+posted: 2023-03-08
+post: true
+thumbnails: nightstand
+summary: The new Nighstand mode shows the state of a charging watch at a glance.
+author: Ed Beroset (beroset)
+---
+
+<p>When a watch is on the charger, it's useful to be able to see the state of charge at a glance.  We've implemented a new feature called *Nightstand mode* that helps with this.</p>
+
+<p>As is often the case with AsteroidOS development, there is often conversation about what new features should be implemented and how they should work.  To help explain to the rest of the community what Nightstand mode might do, and how it might work, the following scenario and requirements descriptions were inserted into a <a href="https://github.com/AsteroidOS/asteroid/issues/151">issue description</a>.</p>
```

Comment:
To keep this an easy read we can probably simplify this quite a bit by just saying something along the line of:

The feature was originally requested in a GitHub issue (link to the specific comment https://github.com/AsteroidOS/asteroid/issues/151#issuecomment-1204035843) and described in these terms: